manifests/nodejs.yml

```diff
@@ -94,87 +94,154 @@ tests/:
             express4: v3.19.0
     waf/:
       test_addresses.py:
-        Test_BodyJson: v2.2.0
+        Test_BodyJson: 
+          '*': v2.2.0
+          nextjs: v4.17.0
         Test_BodyRaw: missing_feature
-        Test_BodyUrlEncoded: v2.2.0
-        Test_BodyXml: v2.2.0
+        Test_BodyUrlEncoded: 
+            '*': v2.2.0
+            nextjs: missing_feature  # request.formData() method not supported yet
+        Test_BodyXml: 
+            '*': v2.2.0
+            nextjs: irrelevant  # Body xml is not converted to JSON in nextjs
         Test_ClientIP: missing_feature
-        Test_Cookies: v2.0.0
+        Test_Cookies:
+          '*': v2.0.0
         Test_FullGrpc: missing_feature
-        Test_GraphQL: v4.16.0
-        Test_Headers: v2.0.0
+        Test_GraphQL:
+          '*': v4.16.0
+          nextjs: irrelevant # nextjs is not related with graphql
+        Test_Headers:
+          '*': v2.0.0
```

Comment:
Turning a simple `Test_Headers: v2.0.0` into this is ew imo, especially when doing it everywhere, it decreases readability

Reply:
You're absolutely right. I modified it to add nextjs but when I remove it, I maintained it as before 🤦